### PR TITLE
Adding AxiStreamCompact.vhd

### PR DESCRIPTION
**Pull request by ruck314**
### Description
- [Copied src from FilMarini:surf:roce fork](https://github.com/slaclab/surf/commit/11a7628a8c835d372f1e4ec8beefb3bc0b4e9486) (Credit to @FilMarini)
  - Required for future RoCEv2 support
- [emacs VHDL beautify](https://github.com/slaclab/surf/pull/1184/commits/04fb1a5fb1e31460b3da3059012c2b44ce080af5)
- [removing isRoCE port (application specific) and switch to standard reset convention](https://github.com/slaclab/surf/pull/1184/commits/00ebb0500fa68ac4180aa9b5ad198e9a4e564518)

### CODE REVIEW

**Review by bengineerd — APPROVED** [review]

**Comment by FilMarini** [comment]
Hi Larry,

just FYI, I just found a bug (like literally 5 minutes ago) concerning this module.

You might wanna wait until a fix before merging the pre-release into an actual release.

Sorry,
Filippo

**Pull request merged**
